packages/Webkul/Product/src/Exceptions/InsufficientProductInventoryException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Webkul\Product\Exceptions;
+
+use Exception;
+use Throwable;
+
+class InsufficientProductInventoryException extends Exception
+{
+    public function __construct(?string $message = null, ?Throwable $previous = null)
+    {
+        parent::__construct(
+            $message ?? trans('product::app.checkout.cart.inventory-warning'),
```

Comment:
I think just leave the class as it is without constructor and add the message in each exception.